models/org.go

```diff
@@ -471,20 +471,17 @@ func GetOwnedOrgsByUserIDDesc(userID int64, desc string) ([]*User, error) {
 // GetOrgsCanCreateRepoByUserID returns a list of organizations where given user ID
 // are allowed to create repos.
 func GetOrgsCanCreateRepoByUserID(userID int64) ([]*User, error) {
-	orgIDs := make([]int64, 0, 10)
-
-	// because xorm do not return all cols, we have to collect IDs first and return user based on them afterwards
-	if err := x.Join("INNER", "`team_user`", "`team_user`.org_id=`user`.id").
-		Join("INNER", "`team`", "`team`.id=`team_user`.team_id").
-		Where("`team_user`.uid=?", userID).
-		And(builder.Eq{"`team`.authorize": AccessModeOwner}.Or(builder.Eq{"`team`.can_create_org_repo": true})).
-		Desc("`user`.updated_unix").GroupBy("`user`.id").Table("`user`").Cols("`user`.id").
-		Find(&orgIDs); err != nil {
-		return nil, err
-	}
+	orgs := make([]*User, 0, 10)
 
-	orgs := make([]*User, 0, len(orgIDs))
-	return orgs, x.In("id", orgIDs).Find(&orgs)
+	return orgs, x.Where("id IN ("+
+		"SELECT `user`.id FROM `user` "+
+		"INNER JOIN `team_user` ON `team_user`.org_id = `user`.id "+
+		"INNER JOIN `team` on `team`.id = `team_user`.team_id "+
```

Comment:
```suggestion
		"INNER JOIN `team` ON `team`.id = `team_user`.team_id "+
```

Reply:
It doesn't particularly matter but it's nice to have the SQL syntax in all capitals.